### PR TITLE
Updated outreach blurb 

### PR DESCRIPTION
**Pull request by mbonsma**
Fixes issue #149.

### CODE REVIEW

File: outreach_announcement.md [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
-http://uoftcoders.github.io/studyGroup/). Also, if you could take a second to fill out [this survey](https://docs.google.com/forms/d/1QIRsnaJjpf5sTV_nMC9KEA44Mz21wuASDxttdgTBj44/viewform?usp=send_form) to let us know what you'd be interested in, that would be a big help, too. Finally, if you'd like updates on what our Study Group is doing, watch our [events repository](https://github.com/UofTCoders/Events) by clicking 'Watch' at the top of our Github repo. 
+Our next meeting will be **event title** on **date** at 6 pm in MP408 (McLennan Physical Laboratories). If you can't make it out but have ideas or questions, please post a thread on our [issue tracker](https://github.com/UofTCoders/studyGroup/issues) or check out the group [website](
+https://uoftcoders.github.io/). Finally, if you'd like to stay informed about upcoming events, you can follow our [Google Calendar](https://calendar.google.com/calendar/embed?src=uoftcoders@gmail.com), sign up for our [email list](https://groups.google.com/forum/#!forum/uoftcoders), or watch our [events repository](https://github.com/UofTCoders/Events) by clicking 'Watch' at the top of our Github repo.
```
Comment (joelostblom) [inline outreach_announcement.md]: Remove "Finally, "?

File: outreach_announcement.md [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+uoftcoders@gmail.com
+
+
+------------------
+
+## Shorter blurb (i.e. GSU newsletter)
+
+UofT Coders: Wednesdays at 6 pm
+
+UofT Coders is a group for researchers and learners to get together, share coding skills, and help each other out with the programming many have to do as part of research. Topics from across the spectrum of computing and research practices are covered. UofT Coders meets on Wednesdays at 6:00 pm in MP408. To receive updates on future sessions, follow the UofTCoders Events repository on Github (https://github.com/UofTCoders/Events) or send an email to uoftcoders@gmail.com.
```
Comment (joelostblom) [inline outreach_announcement.md]: "programming many have to do as part of research." -> "programming we do as part of our research."

File: outreach_announcement.md [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
 
-Most Study Group sessions are hour-long tutorials where one person from our community leads a hands-on demonstration of a tool, package or piece of code, while everyone else follows along. Also, we get together once in a while for open coworking sessions to work on our own research coding and get help with questions as they arise. The tutorial sessions are led by volunteers, and the curriculum and schedule is up to us to set. 
+Most UofT Coders sessions are hour-long tutorials where one person from our community leads a hands-on demonstration of a tool, package or piece of code, while everyone else follows along. We also get together for open coworking sessions to work on our own research coding and get help with questions as they arise. The tutorial sessions are led by volunteers, and the curriculum and schedule is up to us to set.
```
Comment (joelostblom) [inline outreach_announcement.md]: "We also get together for open coworking sessions to work on our own research coding and get help with questions as they arise. The tutorial sessions are led by volunteers, and the curriculum and schedule is up to us to set."

->

"These tutorials are led by volunteers, and our members design the curriculum and schedule. We also get together for open coworking sessions to work on our own research coding and get help with questions as they arise."

File: outreach_announcement.md (hunk 1 above)
Comment (lwjohnst86) [inline outreach_announcement.md]: The `[website](` breaks from a newline. Not sure if that affects the link at all.

**Pull request merged**